python/pynq/intf/intf_const.py

```diff
@@ -37,6 +37,8 @@
 
 # Microblaze constants
 BIN_LOCATION = os.path.dirname(os.path.realpath(__file__)) + "/"
+ARDUINO = 3
+INTERFACE_ID = [ARDUINO]
```

Comment:
these are hardcoded values that we need to get rid of somehow.  Can it be part of intf_spec to say the Micbolaze name or index?  This will not port to other boards.